pkgs/applications/networking/davmail/default.nix

```diff
@@ -1,10 +1,10 @@
 { fetchurl, stdenv, jre, glib, libXtst, gtk2, makeWrapper }:
 
 stdenv.mkDerivation rec {
-  name = "davmail-4.7.2";
+  name = "davmail-4.8.0";
   src = fetchurl {
-    url = "mirror://sourceforge/davmail/4.7.1/davmail-linux-x86_64-4.7.1-2416.tgz";
-    sha256 = "196jr44kksb197biz984z664llf9z3d8rlnjm2iqcmgkjhx1mgy3";
+    url = "mirror://sourceforge/davmail/4.7.1/davmail-linux-x86_64-4.8.0-2479.tgz";
```

Comment:
oops, the mirror url seems to wrong, still points to 4.7.1. building works never the less as i'm running 4.8.0. i will change the url, re-test and rebase my commits.